kubernetes/client/api_client.py

```diff
@@ -505,6 +505,21 @@ def select_header_content_type(self, content_types):
 
         content_types = [x.lower() for x in content_types]
 
+        if method and body and method == 'PATCH':
+            # try to select valid content_type
+            if 'application/json-patch+json' in content_types and \
+               'application/merge-patch+json' in content_types:
```

Comment:
what about changing  'application/merge-patch+json' in content_types: to

('application/merge-patch+json' in content_types or 'application/strategic-merge-patch+json' in content_types):

Reply:
Good point. It should be implemented in this way.

Reply:
Anything related to `application/strategic-merge-patch+json` will be a patch maintained by us IIUC